rosserial_client/src/ros_lib/ros/node_handle.h

```diff
@@ -348,8 +348,8 @@ namespace ros {
       }
 
       /* Register a new subscriber */
-      template<typename MsgT>
-      bool subscribe(Subscriber< MsgT> & s){
+      template<typename Subscriber>
+      bool subscribe(Subscriber& s){
```

Comment:
I know I suggested this, but given the other types which exist in this scope, `SubscriberT` might be a less confusing name.